docs/writing-stories/introduction.md

```diff
@@ -84,7 +88,7 @@ Use the _named_ exports of a CSF file to define your component’s stories. We r
 
 <CodeSnippets
   paths={[
-    'react/button-story.with-hooks.js.mdx',
+    'react/button-story.with-hooks.js.mdx'
```

Comment:
Just a small item here, if you could leave in the dangling comma, I'd appreciate it. It's just for consistency's sake.

Reply:
Fixed.